app/controllers/payments_controller.rb

```diff
@@ -0,0 +1,4 @@
+class PaymentsController < ApplicationController
```

Comment:
Going to restrict controller access to the new ui similar to 
https://github.com/brave-intl/publishers/pull/1495/files#diff-35a55926102be0f7565d1bea8f1207e5R374

Reply:
https://github.com/brave-intl/publishers/pull/1555/commits/886a4d60d3f1eea48ce955bdfd09b5936da2ffef